karma.requirejs.conf.js

```diff
@@ -0,0 +1,70 @@
+'use strict';
+
+module.exports = function (config) {
```

Comment:
There's a lot of duplication in this file as well. How about creating `karma-shared.conf.js` that will have options shared by both configs and then requiring it here and just extending by proper options? See e.g. how Angular does it:
https://github.com/angular/angular.js/blob/master/karma-jquery.conf.js
https://github.com/angular/angular.js/blob/master/karma-shared.conf.js
etc.